datalab/datalab_session/tests/test_utils.py

```diff
@@ -0,0 +1,61 @@
+import pathlib as pl
+
+from django.test import TestCase
+
+from datalab.datalab_session.util import *
+from django.conf import settings
+
+# extending the TestCase class to include a custom assertions
+class TestCaseCustom(TestCase):
+    def assertIsFile(self, path):
+        if not pl.Path(path).resolve().is_file():
+            raise AssertionError("File does not exist: %s" % str(path))
+
+class S3UtilTestClass(TestCase):
+
+  def test_bucket(self):
+    # Test that the bucket is set to the correct value
+    self.assertEqual(settings.DATALAB_OPERATION_BUCKET, 'datalab-operation-output-lco-global')
+
+  def test_key_exists(self):
+    self.assertFalse(key_exists('nonexistent_test_key'))
```

Comment:
You don't want tests connecting to S3 at all - they shouldn't need to connect to production services. So I would just leave out S3 tests. If you really want to, I would either have a test minio bucket that has test data to test with, but that can be problematic since minio doesn't behave exactly the same as S3 always.